src/Elliptic/BoundaryConditions/BoundaryConditionType.hpp

```diff
@@ -0,0 +1,46 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <ostream>
+
+/// \cond
+namespace Options {
+class Option;
+template <typename T>
+struct create_from_yaml;
+}  // namespace Options
+/// \endcond
+
+namespace elliptic {
+
+/// Identify types of boundary conditions for elliptic equations
+enum class BoundaryConditionType {
+  /// Dirichlet boundary conditions like \f$u(x_0)=u_0\f$
+  Dirichlet,
+  /// Neumann boundary conditions like \f$n\cdot\nabla u(x_0)=v_0\f$, where
```

Comment:
I'm not sure if the elliptic code uses `nabla` or `partial_i`, this should be consistent with the rest of the elliptic code. The evolution code uses `partial_i` to avoid any ambiguities with GR

Reply:
It should be using `partial_i` for partial derivatives and be careful about `nabla`. Changed.